projects/storefrontlib/src/lib/store-finder/components/store-finder-list/store-finder-list.component.html

```diff
@@ -1,73 +1,11 @@
-                <cx-store-finder-map #storeMap [locations]="locations.stores" (selectedStoreItem)="selectStoreItemList($event)"></cx-store-finder-map>
-            </div>
-        </div>
-
-        <!-- mobile tabs for column set only -->
-
-        <div *ngIf="locations?.stores" class="cx-store-finder-list__column-set-mobile">
-            <ngb-tabset justify="center">
-                <ngb-tab>
-                    <ng-template ngbTabTitle>
-                        <div class="cx-store-finder-list__column-set-mobile-title">
-                            List View
-                        </div>
-                    </ng-template>
-                    <ng-template ngbTabContent>
-                        <div class="cx-store-finder-list__address-column">
-                            <ol class="cx-store-finder-list__list">
-                                <li *ngFor="let location of locations?.stores; let i = index;" id="{{'item-' + i}}"
-                                    [ngClass]="{'cx-store-finder-list__selected-store-item' : selectedStore === i}"
-                                    class="cx-store-finder-list__list-items cx-store-finder-list__list-items--ordered">
-                                    <cx-store-finder-list-item [location]="location" [locationIndex]="i"
-                                        (storeItemClick)="centerStoreOnMapByIndex($event)"></cx-store-finder-list-item>
-                                </li>
-                            </ol>
-                        </div>
-                    </ng-template>
-                </ngb-tab>
-                <ngb-tab>
-                    <ng-template ngbTabTitle>
-                        <div class="cx-store-finder-list__column-set-mobile__tab-title">
-                            Map View
-                        </div>
-                    </ng-template>
-                    <ng-template ngbTabContent>
-                        <div class="cx-store-finder-list__map-column">
-                            <cx-store-finder-map #storeMap [locations]="locations.stores" (selectedStoreItem)="selectStoreItemList($event)"></cx-store-finder-map>
-                        </div>
-                    </ng-template>
-                </ngb-tab>
-            </ngb-tabset>
-        </div>
-
-        <!-- mobile tabs end -->
-
-        <div *ngIf="locations?.stores" class="row">
-            <div class="col-md-6 offset-md-3 cx-store-finder-list__pagination">
-                <cx-pagination [pagination]="locations.pagination" (viewPageEvent)="viewPage($event)"></cx-pagination>
-            </div>
-        </div>
-
-        <div *ngIf="!locations?.stores" class="row">
-            <div class="col-md-12 cx-store-finder-list__not-found">
-                No Stores Found.
-            </div>
+<div *ngIf="!(isLoading$ | async) && (locations$ | async) as locations; else loading">
+    <cx-store-finder-display-list [locations]=locations></cx-store-finder-display-list>
```

Comment:
I just had the craziest idea. How about `store-finder-list` becomes `store-finder-results` and `store-finder-display-list` becomes `store-finder-list`?